blueprints/addon/files/.github/workflows/ci.yml

```
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request: {}
+
+jobs:
+
+  lint:
+    name: "Lint"
+    env:
+      CI: true
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v2
+      with:
+        node-version: 12.x
+
+    - uses: actions/cache@v2
+      with:
+        path: <%= yarn ? '${{ steps.yarn-cache-dir-path.outputs.dir }}' : '~/.npm' %>
+        key: ${{ runner.os }}-modules-${{ hashFiles('<%= yarn ? '**/yarn.lock' : '**/package-lock.json' %>') }}
```

Let's leverage the new `cache` option added to `setup-node` (in https://github.com/actions/setup-node/releases/tag/v2.2.0):

```suggestion
    - name: Install node
      uses: actions/setup-node@v2
      with:
        node-version: 12.x
        cache: <%= yarn ? 'yarn' : 'npm' %>
```